doc/ref_model/chapters/chapter01.md

```diff
@@ -24,33 +24,34 @@ To help guide the reader, a glossary [Reference Model Terminology](../../tech/gl
 This section introduces the high-level principles of infrastructure abstraction and profiling that will be used in context of this document.
 
 1. A top-level objective of the CNTT is to build a single, overarching Reference Model with the smallest number of Reference Architectures tied to it as is practical. Two principles are introduced in support of these objectives:
-    - **Minimize Architecture proliferation by stipulating compatible features be contained within a single Architecture as much as possible:**
+    - **Minimise Architecture proliferation by stipulating compatible features be contained within a single Architecture as much as possible:**
       - Features which are compatible, meaning they are not mutually exclusive and can coexist in the same cloud infrastructure instance, shall be incorporated into the same Reference Architecture. For example, IPv4 and IPv6 should be captured in the same Architecture, because they don't interfere with each other
       - Focus on the commonalities of the features over the perceived differences. Seek an approach that allows small differences to be handled at either the low-level design or implementation stage. For example, assume the use of existing common APIs over new ones.
 
     - **Create an additional Architecture only when incompatible elements are unavoidable:**
       - Creating additional Architectures is limited to when incompatible elements are desired by Taskforce members. For example, if one member desires KVM be used as the hypervisor, and another desires ESXi be used as the hypervisor, and no compromise or mitigation* can be negotiated, the Architecture could be forked, subject to review and vote to approve by the CNTT technical Working Group, such that one Architecture would be KVM-based and the other would be ESXi-based.
```

Comment:
Minor nit: I see the asterisk was removed from the paragraph following this one.  I believe it was intended to be a footnote on the word mitigation in this paragraph.  Removing the asterisk below should also trigger removal of the asterisk in this paragraph.  
```suggestion
      - Creating additional Architectures is limited to when incompatible elements are desired by Taskforce members. For example, if one member desires KVM be used as the hypervisor, and another desires ESXi be used as the hypervisor, and no compromise or mitigation can be negotiated, the Architecture could be forked, subject to review and vote to approve by the CNTT technical Working Group, such that one Architecture would be KVM-based and the other would be ESXi-based.
```